### PR TITLE
Added phylotree module for tree visualisation.

### PR DESCRIPTION
**Pull request by guignonv**
#

Issue #

### CODE REVIEW

**Comment by bradfordcondon** [comment]
* please keep modules in alphabetical order.  This should be between  ND Genotypes and Tripal Fancy Fields.
 * I dont know how the footnotes style links were supposed to display, but i dont think it is as intended.

screen shot 2019-01-23 at 3 51 49 pm

I would suggest just formatting them as links: if you prefer the footnotes, RST supports that too http://docutils.sourceforge.net/docs/user/rst/quickref.html#footnotes but i'd strongly suggest links because then your links are "contained" in your module description.  I've offered the code below.


```

This extension provides a simple file formatter for `newick <http://evolution.genetics.washington.edu/phylip/newicktree.html>`_ tree files using
the Javascript library `phylotree <https://github.com/veg/phylotree.js/tree/master>`_ for display.

```

**Comment by bradfordcondon** [comment]
also if you wouldnt mind also fixing the order of the other modules to be alphabetical in this PR I'd appreciate it.
 https://github.com/tripal/tripal/pull/831#issuecomment-456859547

sorry for not catching it before merging.

This is for both Brapi and Migrate Chado.

**Comment by guignonv** [comment]
Order updated. ;)

**Comment by bradfordcondon** [comment]
Thanks.  how about the link formatting?

**Comment by guignonv** [comment]
I'm new to read-the-docs so I don't know much about formatting. Is it better this way?

**Comment by bradfordcondon** [comment]
its still going to visualize with those brackets.  What about the code i suggested in https://github.com/tripal/tripal/pull/833#issuecomment-456962406 ?

**Comment by guignonv** [comment]
Well, I solved the potential link problem differently. And about #833, order was already fixed in previous changes. ;)

**Comment by bradfordcondon** [comment]
ok thanks!

I notice that your link still wont format correctly :P   You need a closing `>` on both links. you also need a space between the text and the link `Newick <` not `Newick<` otherwise it wont render.
I'm just going to fix it and merge.  
```
`Newick<http://evolution.genetics.washington.edu/phylip/newicktree.html`__
```

you can see GitHub rendering it here:  https://github.com/tripal/tripal/blob/39d09c8294c42fa6a8438e9aa6a64f5b9f3f8924/docs/extensions/visualization.rst#phylotree



edit- i cant comit directly because its on your fork, so i've made the suggestions below via github

File: docs/extensions/visualization.rst [hunk 1]
```diff
@@ -35,6 +35,15 @@ This module provides support and visualization of genotypic data stored in a mod
 `Documentation <https://nd-genotypes.readthedocs.io/en/latest/>`__
 `Repository <https://github.com/UofS-Pulse-Binfo/nd_genotypes>`__
 
+Phylotree
+-------------------
+
+This extension provides a simple file formatter for `Newick<http://evolution.genetics.washington.edu/phylip/newicktree.html`__ tree files using
```
Comment (bradfordcondon) [inline docs/extensions/visualization.rst]:
```suggestion
This extension provides a simple file formatter for `Newick <http://evolution.genetics.washington.edu/phylip/newicktree.html>`__ tree files using
```

File: docs/extensions/visualization.rst [hunk 2]
```diff
@@ -35,6 +35,15 @@ This module provides support and visualization of genotypic data stored in a mod
...
+the Javascript library `Phylotree <https://github.com/veg/phylotree.js/tree/master`__ for display.
```
Comment (bradfordcondon) [inline docs/extensions/visualization.rst]:
```suggestion
the Javascript library `Phylotree <https://github.com/veg/phylotree.js/tree/master>`__ for display.
```

**Comment by guignonv** [comment]
Thanks! Sorry, I don't have a tool to preview the mistakes I made. :-s

**Comment by bradfordcondon** [comment]
No problem, thanks for working with me.  Sorry for being a pain, it's important to me that the handbook has a consistent look :)

screen shot 2019-01-25 at 3 03 05 pm

**Comment by spficklin** [comment]
Thanks @guignonv for updating the docs!!

**Pull request merged**